epub33/rs/index.html

```diff
@@ -1242,11 +1242,11 @@ <h4>The <code>rendition:page-spread-*</code> Properties</h4>
 								><code>page-progression-direction</code></a> [[EPUB-33]] &#8212; when it lacks a
 							<code>rendition:page-spread-*</code> property value.</p>
 
-					<p id="page-spread-flow" data-tests="#page-layout-both-flow">If the reflowable spine item has a
+					<p id="page-spread-both" data-tests="#page-layout-both-spread">If the reflowable spine item has a
 							<code>rendition:page-spread-*</code> specification, it MUST be honored (e.g., by inserting a
 						blank page).</p>
 
-					<p>Similarly, when a pre-paginated spine item follows a reflowable one, the pre-paginated one SHOULD
+					<p data-tests="#page-layout-both-spread">Similarly, when a pre-paginated spine item follows a reflowable one, the pre-paginated one SHOULD
```

Comment:
It's not clear whether this is testing the SHOULD (this line) or the MUST (line 1252), since they're in the same `<p>` tag.

Can you wrap these in `<span>` tags, or, more boldly, break it up into multiple paragraphs to match 1238 and 1245, and then tag them with data-tests as appropriate?

Please also make sure that each span or paragraph has an anchor, and reference those anchors from the tests. Thanks!